_posts/2022-06-08-filesystem-integrity.adoc

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "Enabling Integrity on Filesystem realms"
+date: 2022-06-08
```

Comment:
The date in the file name and the date here will need to be updated to today's date.